network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -312,106 +134,149 @@ void prepareStatements() {
      * Get all networks infos.
      */
     public List<NetworkInfos> getNetworksInfos() {
-        SimpleStatement simpleStatement = selectFrom(NETWORK).columns(UUID_STR, ID_STR)
-                .whereColumn(VARIANT_NUM).isEqualTo(literal(Resource.INITIAL_VARIANT_NUM))
-                .allowFiltering()
-                .build();
-        try (ResultSet resultSet = session.execute(simpleStatement)) {
-            List<NetworkInfos> networksInfos = new ArrayList<>();
-            for (Row row : resultSet) {
-                networksInfos.add(new NetworkInfos(row.getUuid(0), row.getString(1)));
+        try (var connection = dataSource.getConnection()) {
+            try (var stmt = connection.createStatement()) {
```

Comment:
I think it's good practice to use preparedStatement anyway here ?

Reply:
Only when there is parameters. In that case there is not and prepared statement just add an overhead.